packages/lib-classifier/src/store/Classification.js

```diff
@@ -46,6 +47,43 @@ const Classification = types
     }),
     metadata: types.maybe(ClassificationMetadata)
   })
+  .views(self => ({
+    get toSnapshot () {
```

Comment:
Is there any chance this will have an invalid cached version of a classification since this is using a getter? We might not want or need this to be pre-computed.

Reply:
i don't know enough about views to say. I thought they were computed each time the model changed?

Reply:
My understanding is that it will be recomputed if one of the state values that the derivations uses changes, but I find this to be risky in this case. We only really want the final state of the classification to be retrieved here. I think all we have to do is remove `get` and this will no longer be cacheable. My comment applies for all the models using this pattern.

Reply:
Good point. 🤔